docs/internal/input-validations-framework-readme.md

```diff
@@ -0,0 +1,56 @@
+# Description
+
+This is an internal readme to explain how to use the validation constraints framework to detect
+errors at build time rather than run time.
+
+## What can you use with the framework
+
+You will find in the constraints-utils.ts file the generic calls for the following classes if you properly import them from the constraints-utils.ts file.
```

Comment:
what do you mean by "generic calls"? Just state something like:


# How to use the framework

The framework implementation is located in the utils/contraints-utils.ts module: 

```typescript
import * from 'utils/constraints-utils`;
```